appengine/standard/ndb/schema_update/add_entities_handler.py

```diff
@@ -0,0 +1,29 @@
+import webapp2
```

Comment:
imports should be in 3 sections - standard library, third-party, and (optionally) application/package-local names

This should be:

```
import os
import time

import jinja2
import webapp2

import update_schema
```